app-misc/boxes/files/boxes-2.2.0-fix-clang16-build.patch

```diff
@@ -0,0 +1,13 @@
+change function prototype to work with c2x standard. Build fails with clang16 without this
```

Comment:
1. Could you send this upstream and include a link to where you did so at the top of the patch?
2. What about that calloc thing?

Reply:
You mean the QA issue? Gone with 2.2.0

Reply:
Ah, great!